### PR TITLE
Adding Desktop to System.ComponentModel.Annotations

### PR DESCRIPTION
**Pull request by venkat-raman251**

### CODE REVIEW

**Comment by venkat-raman251** [comment]
@weshaggard : Please have a look.

File: src/System.ComponentModel.Annotations/src/System.ComponentModel.Annotations.csproj [hunk 1]
```diff
@@ -9,13 +9,14 @@
     <RootNamespace>System.ComponentModel.Annotations</RootNamespace>
     <AssemblyName>System.ComponentModel.Annotations</AssemblyName>
     <AssemblyVersion>4.1.0.0</AssemblyVersion>
-    <PackageTargetFramework>dotnet5.4</PackageTargetFramework>
```
Comment (weshaggard) [inline src/System.ComponentModel.Annotations/src/System.ComponentModel.Annotations.csproj]: Should be conditioned, and also net46 should be IsPartialFacadeAssembly=true.

File: src/System.ComponentModel.Annotations/src/project.json [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
+        "System.Resources.ResourceManager": "4.0.0",
+        "System.Runtime": "4.0.20",
+        "System.Runtime.Extensions": "4.0.10",
+        "System.Text.RegularExpressions": "4.0.10",
+        "System.Threading": "4.0.10"
+      }
+    },
+    "net46":{
+      "dependencies": {
+        "Microsoft.TargetingPack.NetFramework.v4.6": "1.0.0-rc2-23530",
```
Comment (weshaggard) [inline src/System.ComponentModel.Annotations/src/project.json]: remove "-rc2-23530"

File: src/System.ComponentModel.Annotations/src/System.ComponentModel.Annotations.csproj (hunk 1 above)
Comment (venkat-raman251) [inline src/System.ComponentModel.Annotations/src/System.ComponentModel.Annotations.csproj]: @weshaggard : Updated commit with the changes.

File: src/System.ComponentModel.Annotations/src/System.ComponentModel.Annotations.csproj [hunk 3]
```diff
@@ -60,11 +63,15 @@
     <Compile Include="System\ComponentModel\DataAnnotations\ValidationResult.cs" />
     <Compile Include="System\ComponentModel\DataAnnotations\Validator.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'==''">
```
Comment (weshaggard) [inline src/System.ComponentModel.Annotations/src/System.ComponentModel.Annotations.csproj]: Seems like this ItemGroup should be merged with the one above it.

File: src/System.ComponentModel.Annotations/src/project.json [hunk 4]
```diff
@@ -1,21 +1,27 @@
...
-    "System.Reflection.Extensions": "4.0.0",
-    "System.Resources.ResourceManager": "4.0.0",
-    "System.Runtime": "4.0.20",
-    "System.Runtime.Extensions": "4.0.10",
-    "System.Text.RegularExpressions": "4.0.10",
-    "System.Threading": "4.0.10"
-  },
   "frameworks": {
-    "dnxcore50": {}
+    "dnxcore50": {
```
Comment (weshaggard) [inline src/System.ComponentModel.Annotations/src/project.json]: dnxcore50->dotnet5.4

**Comment by venkat-raman251** [comment]
@weshaggard : Updated with all the changes.

**Comment by weshaggard** [comment]
LGTM ready to merge when CI is green.

**Pull request merged**